litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -612,7 +612,7 @@ spec:
             - name: HUB_BRANCH_NAME
               value: "v2.3.x"
             - name: LITMUS_AUTH_GRPC_ENDPOINT
-              value: "litmusportal-server-service.litmus.svc.cluster.local"
+              value: "localhost"
```

Comment:
would it be better to have 127.0.0.1 rather than localhost? Don't think there might be much difference but if there's some name resolution issue in the pod localhost might not be resolved to the proper IP addr. We need to also check if this change works for both ipv4 and ip6 setups

Reply:
I have made the change to `127.0.0.1` & tested in ipv4 & ipv6 working fine.

Reply:
Awesome!